src/OpenTelemetry/Metrics/Exemplar/AlwaysOffExemplarFilter.cs

```diff
@@ -1,31 +1,17 @@
 // Copyright The OpenTelemetry Authors
 // SPDX-License-Identifier: Apache-2.0
 
-#if EXPOSE_EXPERIMENTAL_FEATURES && NET8_0_OR_GREATER
-using System.Diagnostics.CodeAnalysis;
-using OpenTelemetry.Internal;
-#endif
-
 namespace OpenTelemetry.Metrics;
 
-#if EXPOSE_EXPERIMENTAL_FEATURES
 /// <summary>
 /// An <see cref="ExemplarFilter"/> implementation which makes no measurements
 /// eligible for becoming an <see cref="Exemplar"/>.
 /// </summary>
 /// <remarks>
-/// <inheritdoc cref="Exemplar" path="/remarks/para[@experimental-warning='true']"/>
 /// Specification: <see
 /// href="https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/metrics/sdk.md#alwaysoff"/>.
 /// </remarks>
-#if NET8_0_OR_GREATER
-[Experimental(DiagnosticDefinitions.ExemplarExperimentalApi, UrlFormat = DiagnosticDefinitions.ExperimentalApiUrlFormat)]
-#endif
-public
-#else
-internal
-#endif
-sealed class AlwaysOffExemplarFilter : ExemplarFilter
+internal sealed class AlwaysOffExemplarFilter : ExemplarFilter
```

Comment:
we can now get rid of these classes and optimize further? https://github.com/open-telemetry/opentelemetry-dotnet/blob/main/src/OpenTelemetry/Metrics/AggregatorStore.cs#L951-L952

Similar to [Trace sampler special casing](https://github.com/open-telemetry/opentelemetry-dotnet/blob/main/src/OpenTelemetry/Trace/TracerProviderSdk.cs#L224-L234) but we know Filter is only of 3 types, so can do the extreme.

Reply:
This cleanup can be done in a follow up, to keep PR size smaller.